ResourceStatusSystem/Client/SiteStatus.py

```diff
@@ -43,7 +43,6 @@ def __init__( self ):
     self.__opHelper = Operations()
     self.rssFlag = ResourceStatus().rssFlag
     self.rsClient = ResourceStatusClient()
-    self.wmsAdministrator = RPCClient( 'WorkloadManagement/WMSAdministrator' )
```

Comment:
In general, it is safer to create RPCClient just before it is used, the overhead is minimal if any. It is also safer with respect to multithreading